dataframe/dataframe.go

```diff
@@ -887,15 +898,23 @@ func LoadRecords(records [][]string, options ...LoadOption) DataFrame {
 	if cfg.hasHeader && len(records) <= 1 {
 		return DataFrame{Err: fmt.Errorf("load records: empty DataFrame")}
 	}
+	if len(cfg.names) > len(records[0]) {
```

Comment:
Sure we can do that, I just followed how it works with pandas as I expect people are kind of used to that. But if you want to only allow this options if it supplies names for all columns, just let me know. Then I'll change it.